docs/source/admin/traffic_ops/using.rst

```diff
@@ -296,30 +296,33 @@ The fields in the Delivery Service view are:
 +--------------------------------------------------+---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------+
 |                       Name                       |                                                                                                     Description                                                                                                     |
 +==================================================+=====================================================================================================================================================================================================================+
-| XML ID                                           | A unique string that identifies this delivery service.                                                                                                                                                              |
+| Active                                           | When this is set to no Traffic Router will not serve DNS or HTTP responses for this delivery service.                                                                                                               |
```

Comment:
When this is set to false ?

Reply:
this is a work in progress. i just restructured the table to ensure it captures all the fields. will need others to help fill in the blanks...

Reply:
oh, nm. i see what you're asking. don't worry about the content yet. it will be rewritten